### ISSUE TITLE
Who has the authority to change the name of ISIS?

### ISSUE THREAD

**Issue by jessemapel** [USGS-Astrogeology/ISIS_TC#148]
Changing to something other than ISIS has come up multiple times in the past and is coming back up again in the DEIA (Diversity, Equity, Inclusion, and Accessibility) council at the ASC. This group is just ASC employees and not a part of the management.

There is one big question that needs to be answered before even deciding if the name will change: Who has the authority to change the name of ISIS? Is it the ISIS TC? Is it the ASC management? Is is the ISIS developers? Is it anyone who is interested in doing the work?

**Comment by rbeyer** [comment]
If ISIS is a community project, and that community (including the large number of USGS developers that are paid to contribute to it) is adequately represented by the ISIS TC, then it seems to me that the ISIS TC should have the authority to change the name of the project, if they decide to do so.

**Comment by jlaura** [comment]
I do not believe that the TC, in isolation, has the authority to unilaterally change any name other than it's own. I believe that the ISIS TC has the authority to propose a change in the name of the project to the contributors on the ISIS project under the auspices of a technical policy change. 

If a name change is endorsed by the TC, the contributors on the ISIS project would need to determine the adoption cadence and iterate with the ISIS TC as necessary to ensure that an appropriate migration plan has been put forth. It is important that the burden of changes proposed by the TC be shared between the TC and the ISIS contributor community. In this instance, the burden can not be borne solely by the contributors on the ISIS project (and by extension the subset of contributors that are the ASC development group) as this has the potential to be a significant impact to the project.

**Comment by rbeyer** [comment]
@jlaura, I think we are in agreement, but I think your language above is imprecise.

I think that under the terms of the [charter](https://github.com/USGS-Astrogeology/ISIS_TC/blob/master/TC-Charter.md#section-4-responsibilities-of-the-tc) the ISIS TC has the authority to change the name, if it chooses to do that.  

Of course, given our policy of lazy consensus, this means that we must have consensus on the ISIS TC to arrive at that decision.  And in order to get there, I think that there is a very large process that we must undergo (many of the items which you detail).  Firstly, we need a clean description of what's at issue, what are the potential remedies, and what are the potential impacts.  My understanding is that the DEIA council at the ASC (of which some TC members also participate) are working to consolidate and present this case. 

At that point, the TC needs to determine (via consensus) whether to explore the matter further or mark it "do not fix."
If the TC decides to explore further, then a more complete evaluation of the impacts must be undertaken, which is a non-trivial amount of effort and work.  The ASC developers are not funded to undertake that action at this time, and so that activity would have to fall on the shoulders of interested volunteers, which may delay the evaluation.

Only with all of that information before us, could the TC even attempt to determine what action to take, and it could only do so with full consensus.  Since the ISIS contributors (including ASC) are well-represented on the TC, I find it unlikely that the ISIS TC could do anything "unilaterally."  It isn't a majority-rules system, it is a consensus-based system.

So to return to the very narrow question of "Who has the authority to change the name of ISIS?" from the OP and the title of this Issue, I hope we can agree that the answer is "the ISIS TC."   However, even one member of the ISIS TC (which broadly represent the ISIS contributors and their interests) is not in consensus on whether to change the name, then it will not change.
